source/multigrid/mg_tools.inst.in

```diff
@@ -126,6 +126,11 @@ for (deal_II_dimension : DIMENSIONS)
     extract_non_interface_dofs (const DoFHandler<deal_II_dimension> & mg_dof_handler,
                                 std::vector<std::set<types::global_dof_index> > &non_interface_dofs);
 
+
+    template
+    unsigned int
+    max_level_for_coarse_mesh (const Triangulation<deal_II_dimension> & tria);
```

Comment:
Since you declare the template to allow for using both template arguments `dim` and `spacedim`, I would prefer if you would also instantiate it for all sensible combinations.